Byzantium-Hard-Fork-changes.md

```diff
@@ -0,0 +1,14 @@
+<h2>What changes are included in the Byzantium hard fork?</h2>
+<p>The following upgrades are included:</p>
+<ul>
+<li>Addition of ‘REVERT’ opcode, which permits error handling without consuming all gas (<a href="https://github.com/ethereum/EIPs/pull/206">EIP 140</a>)</li>
```

Comment:
If we're going to do release notes, I'd rather link to the EIPs for Byzantium
https://github.com/ethereum/EIPs/blob/master/EIPS/eip-609.md

Reply:
Having said that, I think these release notes should be merged into the `byzantium` branch, not directly into `master`.

Reply:
People may not look at the byzantium branch, it's important to have it in the readme so that people will know that it's not up-to-date, and where they can go to find out about the latest changes, as well as planned changes.

Reply:
#404

Reply:
I edited the readme to refer to the Byzantium branch.